### PR TITLE
Introduce ADOPTERS.md

### PR DESCRIPTION
**Pull request by eriknordmark**
The LF Edge process recommends that there be an ADOPTERS.md for the project. This is an initial version of such a file.

### CODE REVIEW

**Comment by rene** [comment]
@eriknordmark , there is one Yetus' complain: 

`Error: markdownlint:MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]`

I think  you should remove the last line of the file (blank).

Also, could you please add an entry for this file in the https://github.com/lf-edge/eve/blob/master/docs/mkdocs/mkdocs.yml ?

**Comment by eriknordmark** [comment]
> Also, could you please add an entry for this file in the https://github.com/lf-edge/eve/blob/master/docs/mkdocs/mkdocs.yml ?

@rene We don't currently list any of the project markdowns (CONTRIBUTING.md, MAINTAINERS.md) nor the README.md at the top directory. Should I create a new category "Project information" and put all of them under that in mkdocs.yml?

**Comment by rene** [comment]
> > Also, could you please add an entry for this file in the https://github.com/lf-edge/eve/blob/master/docs/mkdocs/mkdocs.yml ?
> 
> @rene We don't currently list any of the project markdowns (CONTRIBUTING.md, MAINTAINERS.md) nor the README.md at the top directory. Should I create a new category "Project information" and put all of them under that in mkdocs.yml?

Indeed, yeah, "Project Information" section sounds good to me....

File: docs/mkdocs/mkdocs.yml [hunk 1]
```diff
@@ -17,6 +17,12 @@ theme:
 extra:
   version: 2.0
 nav:
+  - 'Project Information':
```
Comment (rene) [inline docs/mkdocs/mkdocs.yml:20]: @eriknordmark , now you have some yetus error here: 

`::group::docs/mkdocs/mkdocs.yml:::error: file=docs/mkdocs/mkdocs.yml,line=21,col=5::21:5: [indentation] wrong indentation: expected 6 but found 4`

**Review by deitch — APPROVED** [review]
Looks fine. I assume that you spoke with the named companies and got their approval?

File: pkg/pillar/cmd/downloader/download.go [hunk 2]
```diff
@@ -179,7 +179,7 @@ func download(ctx *downloaderContext, trType zedUpload.SyncTransportType,
 		}
 	}
 
-	var respChan = make(chan *zedUpload.DronaRequest)
+	var respChan = make(chan *zedUpload.DronaRequest, 1)
```
Comment (rene) [inline pkg/pillar/cmd/downloader/download.go:182]: @eriknordmark , change probably forgotten from another branch?

**Comment by eriknordmark** [comment]
> Looks fine. I assume that you spoke with the named companies and got their approval?

Yes, but indirectly via folks at ZEDEDA.

**Pull request merged**